client/components/likes.jsx

```diff
@@ -3,22 +3,42 @@ import React from 'react';
 class Likes extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      liked: false,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(prevState => ({
+      liked: !prevState.liked,
+    }));
   }
 
   render() {
+    /* * * * * * Styling * * * * * */
     const likeStyle = {
       position: 'absolute',
       width: 70,
       height: 25,
       bottom: 30,
       left: 10,
-      border: '2px solid red',
-      borderRadius: '8px',
+      textAlign: 'center',
+      backgroundColor: 'white',
+      border: '1px solid lightgrey',
     };
+    /* * * * Conditional rendering * * * */
+    let page;
+    const { liked } = this.state;
+    if (liked === false) {
+      page = '🖤';
+    }
+    if (liked === true) {
+      page = '🧡';
+    }
     return (
-      <div style={likeStyle}>
-        Likes
+      <div>
+        <button type="button" style={likeStyle} onClick={this.handleClick}>{page}</button>
```

Comment:
Does your css here need to be called 'likeStyle', or can we just call it 'style'? The fact that it exists inside your likes.jsx file adds context to the object itself, and it's not being passed to any other react components. Sorry if this seems nitpicky.